src/java/htsjdk/variant/variantcontext/GenotypesContext.java

```diff
@@ -39,7 +40,9 @@
 /**
  * Represents an ordered collection of Genotype objects
  */
-public class GenotypesContext implements List<Genotype> {
+public class GenotypesContext implements List<Genotype>, Serializable {
```

Comment:
Have modified the tests so that they now detect the issue with `LazyParser` not being `Serializable`. Will work on a fix. Nice catch!